members/diwakar-gupta.md

```diff
@@ -0,0 +1,4 @@
+name: Diwakar Gupta
+
```

Comment:
there's no need for redundant line breaks, please remove them

Reply:
done